libs/ui/elements/src/lib/markdown-parser/markdown-parser.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core'
+import { marked } from 'marked'
+
+@Component({
+  selector: 'gn-ui-markdown-parser',
+  templateUrl: './markdown-parser.component.html',
+  styleUrls: ['./markdown-parser.component.css'],
```

Comment:
I also forgot to point out that we should always use `changeDetection: ChangeDetectionStrategy.OnPush,` to make sure that the change detection does not run too often